docs/pages/workflow/configuration.mdx

```diff
@@ -1,18 +1,18 @@
 ---
-title: Configure with app.json/app.config.js
-description: Learn about what is app config and how you can dynamically use it by customizing it.
+title: Configure with app config
+description: Learn about what app.json/app.config.js/app.config.ts files are and how you can customize and use them dynamically.
 ---
 
 import PossibleRedirectNotification from '~/components/plugins/PossibleRedirectNotification';
 import { Terminal } from '~/ui/components/Snippet';
 
 <PossibleRedirectNotification newUrl="/versions/latest/config/app/" />
 
-The app config (**app.json**, **app.config.js**, **app.config.ts**) is used for configuring how a project loads in [Expo Go](/get-started/expo-go/), [Expo Prebuild](/workflow/prebuild) generation, and the OTA update manifest. You can think of this as an `index.html` but for React Native apps.
+The app config (**app.json**, **app.config.js**, **app.config.ts**) is used for configuring [Expo Prebuild](/workflow/prebuild) generation, how a project loads in [Expo Go](/get-started/expo-go/), and the OTA update manifest. You can think of this as an `index.html` but for React Native apps.
```

Comment:
```suggestion
The app config (**app.json**, **app.config.js**, **app.config.ts**) is used for configuring [Expo Prebuild](/workflow/prebuild) generation, how a project loads in [Expo Go](/get-started/expo-go/), and the OTA update manifest. You can think of this as an **index.html** but for React Native apps.
```

Unrelated to this PR, since this is a filename, we should make it bold.